content/rdi/quickstart/write-behind-guide.md

```diff
@@ -229,6 +241,105 @@ To use the metrics you can either:
 
 - Scrape the metrics using RDI Prometheus exporter
 
+### Read-through jobs
+
+Read-through jobs are a mandatory part of the read-through pipeline configuration.
+
+In the jobs directory, which is located parallel to the config.yaml file, you should include a YAML file for each key pattern you want to read into the cache from a downstream database table in the event of a cache miss.
+
+You can choose to name the YAML file based on the database table name or use another naming convention.
+
+Job definition has the following structure:
+
+```yaml
+source:
+  retries: 1
+  redis:
+    trigger: read-through
+    key_pattern: emp:*
+  keys:
+    delimiter: ":"
+    # alternatively use expression with RegExp
+    fields:
+    - first_name: 1
+    - last_name: 2
+  connection: my-oracle
+  db: test
+  schema: test
+
+  # One of these should be provided:
+  # - table with columns
+  # - sql
+
+  #table: EMP
+  #columns:
+  # - first_name
+  # - last_name
+  # - birth_date
+
+  # bind variables are marked with a colon and must be defined in the `keys` property.
+  sql: |
+    SELECT emp.*, address.*, kids.*
+    FROM emp
+    LEFT JOIN address ON emp.employee_id = address.employee_id
+    LEFT JOIN kids ON emp.employee_id = kids.employee_id
+    WHERE emp.employee_fname= :first_name AND emp.employee_lname = :last_name;
+
+transform:
+  - uses: rename_field
+    with:
+      fields:
+        - from_field: employee_fname
+          to_field: first_name
+        - from_field: employee_lname
+          to_field: last_name
+output:
+  expire: 100
+```
+
+#### The redis section
+
+This section describes the Redis key-pattern and job type:
+
+- The `key_pattern` attribute for the key-pattern that would trigger the read-through job.
+- The `trigger` attribute: indicates it is a read-through and not a write-behind job, using the `read-through` value.
+
+#### The keys section
+
+This section has the following attributes:
+
+- The `delimiter` attribute: specifies a delimiter for tokenizing the Redis key. These tokens are used with the `fields` attribute (see below).
+- Alternatively, specify the `expression` attribute. In this case, the regular expression will be used to tokenize the the Redis key. for example, `expression: "(.+):(\\d+)"` will tokenize the key to anything before the colon and a number after the colon.
+- The `fields` attribute is a map of field name and its offset in the array of tokens generated by the `expression` or the `delimiter` attributes.
+
+#### Downstream database information
+
+The `source` section must specify the following attributes:
+
+- `connection` - A connection alias, referring to a connection section in the `config.yaml` file.
```

Comment:
to a --> to the

Reply:
still

Reply:
to a - it is reffering to specific connection

Reply:
- `connection` - A connection alias, referring to connection in the `config.yaml` file.
I think:
- `connection` - A connection alias which listed under connections section in the `config.yaml` file.